sale_order_move_warehouse/views/sale_order_views.xml

```diff
@@ -0,0 +1,18 @@
+<odoo>
+
+    <record id="sale_order_view_form" model="ir.ui.view">
+        <field name="name">sale.order</field>
+        <field name="model">sale.order</field>
+        <field name="priority">999</field>
+        <field name="inherit_id" ref="sale.view_order_form"/>
```

Comment:
This inheritance is technically wrong, because `warehouse_id` field in view is actually added by a different view, so we should inherit from the right one, could you check?